### ISSUE TITLE
Interop: Document API features supported by 3 engines (Part #2)

### ISSUE THREAD

**Issue by teoli2003** [openwebdocs/project#153]
### Problem statement

There are missing docs in MDN Web/API documentation. We want to increase the coverage here to guarantee that each feature supported by three engines is documented.

There are three groups of missings docs:

1. Numerous features related to HTML and SVG specs that often (but not always) are deprecated or properties reflecting attributes. We don't have them documented because we never prioritized their writing.
2. Missed features scattered on the rest of the specs.** They have been missed during the documentation efforts over the years.
3. We also have a few WebGL, Geometry Interfaces, and WebAssembly features that are not individually documented (like [`WebGLRenderingContext.uniform[1234][fi][v]`](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/uniform) that are documented, on purpose, in one page, or under another hierarchy (like WebAssembly).

Point 2 was solved last Quarter in #152.

### Proposed solutions

The goal of this project is to 
- **Start adding the missing features under 1**:
  - Document all non-HTML*Element features of the HTML spec
  - Document all features related to Constraint Validation
- **Maintain Point 2**:
  - Document all new features matching the criterion in #152 (Non-HTML, Non-SVG interface members supported by 3 engines)

### Task list

- [ ] Document the 105 missing features as listed [here](https://docs.google.com/spreadsheets/d/1qsZGV2-OOAScID-UtTI33IH5Ltc7vpcTccAS2FjBcAk/edit#gid=831294920). (Fix point 2.)

### Priority assessment

- **Effort**: 105 pages means about 12 weeks of work
- **Dependencies**: None.
- **Community enablement**: Yes, this has the potential to attract significant community help 
- **Momentum**: Yes, the mdn-gaps website allows us to detect missing documents easily
- **Enabling learners**: Yes, HTML*Element are APIs that are used a lot by beginners
- **Enabling professionals**: Yes, professionals will be sure to have all features they can use documented 
- **Underrepresented topics / Ethical web**: N/A
- **Operational necessities**: None
- **Addressing needs of the web industry**: MDN would be the only place that can claim to have all API supported by the three engines documented – this batch will terminate interfaces that don't start with HTML or SVG.; with this batch, we also lay the basis to support "Interop-2023-html-forms", that will likely be the focus for the 2nd part of the year.

### More information

This project aims at creating good quality new documents but doesn't aim at fixing current documents.

This project doesn't aim at documenting all `HTML*Element` and `SVG*Element` features, but to unblock community members so they can help us.

The choice of documenting features supported by 3 engines has been taken in order to keep this project small enough so it can be done.

**Comment by teoli2003** [comment]
Thank you!

**Comment by Elchi3** [comment]
Closing this as Jean-Yves is no longer involved with this project. We also created https://openwebdocs.github.io/web-docs-backlog/ to track missing reference documentation. I will create a new issue to work on driving down the missing pages further according to our new backlog dashboard.
